build.gradle

```diff
@@ -22,9 +22,12 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.junit.jupiter:junit-jupiter:5.10.0'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation "org.springframework.boot:spring-boot-starter-web"
+    implementation "org.junit.jupiter:junit-jupiter:5.10.0"
+    implementation "org.springframework.boot:spring-boot-starter-security"
+    implementation "org.springframework.boot:spring-boot-starter-test"
+    testImplementation "org.springframework.boot:spring-boot-starter-test"
```

Comment:
also why did you added this package?

Reply:
In Line 26: JUnit Jupiter API is a program which is used to write and execute test cases.
    Line 27: This package is used to add the relevant dependencies into our spring boot project.
    Line 28: This dependencies add functionality to run our test due to spring boot project.
    Line 29: This works as same as the previous one but the difference is that it only execute during testing time not at the testing time.